fe/src/main/java/org/apache/doris/transaction/TransactionIdGenerator.java

```diff
@@ -30,7 +30,7 @@
     
     private long nextId = NEXT_ID_INIT_VALUE;
     // has to set it to an invalid value, then it will be logged when id is firstly increment
-    private long batchEndId = -1;
+    private long batchEndId = 0;
```

Comment:
Does it work? nextId starts with NEXT_ID_INIT_VALUE, which is 1000. How can 0 be generated?